scripts/mpv-config

```diff
@@ -20,11 +20,18 @@ case "$PKG_CONFIG_PATH" in
     ;;
 esac
 
-# add missing private dependencies from libass.pc
-# this is necessary due to the hybrid static / dynamic nature of the build
-export LDFLAGS="$LDFLAGS $(pkg-config --libs fontconfig harfbuzz fribidi)"
+if [ "$BUILDSYSTEM" = "meson" ]; then
+    echo Using mpv options: "$@"
```

Comment:
Why is this line duplicated? I think it can be shared before the if/else ?

Reply:
I could clean this part up some more.